src/hb-ot-os2-table.hh

```diff
@@ -59,7 +59,7 @@ struct OS2V1Tail
 
 struct OS2V2Tail
 {
-  bool has_data () const { return this != &Null (OS2V2Tail); }
+  bool has_data () const { return sxHeight || sCapHeight; }
```

Comment:
This actually fixes a bug since previously the V2Tail was coming after the OS2 table and as such wouldn't compare equal to Null(OS2VTail).